Update for student

german34 opened the pull request:
Didn't get any errors for this. Let me know if you do. For set gpa I didnt add a parameter since gpa is calculated based on the grades for the courses.

german34 committed:
Update for student

Didn't get any errors for this. Let me know if you do. For set gpa I didnt add a parameter since gpa is calculated based on the grades for the courses.

JSextonn commented:
I can tell just by looking at that code, that its going to give errors.

german34 commented:
Yeah I didn't use the right one. For some reason my file didn't save
correctly when I did it. I also keep getting a syncing error.

On Fri, Feb 17, 2017 at 3:09 AM, Justin Sexton <notifications@github.com>
wrote:

> I can tell just by looking at that code, that its going to give errors.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JSextonn/AgileRecords/pull/5#issuecomment-280595859>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIjVMjM_Pa6b1bXdz0Y2Uo4SaXlWI79Uks5rdWOwgaJpZM4MEBBX>
> .
>



-- 
German Villalobos
german.villalobos0@gmail.com

german34 commented:
My bad disregard the file.

On Fri, Feb 17, 2017 at 3:33 AM, German Villalobos <
german.villalobos0@gmail.com> wrote:

> Yeah I didn't use the right one. For some reason my file didn't save
> correctly when I did it. I also keep getting a syncing error.
>
> On Fri, Feb 17, 2017 at 3:09 AM, Justin Sexton <notifications@github.com>
> wrote:
>
>> I can tell just by looking at that code, that its going to give errors.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/JSextonn/AgileRecords/pull/5#issuecomment-280595859>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AIjVMjM_Pa6b1bXdz0Y2Uo4SaXlWI79Uks5rdWOwgaJpZM4MEBBX>
>> .
>>
>
>
>
> --
> German Villalobos
> german.villalobos0@gmail.com
>



-- 
German Villalobos
german.villalobos0@gmail.com

JSextonn commented:
If you're getting a sync error you may have to just delete the project and reclone it

german34 commented:
Okay. Just did it. I added my previous code to master so it wouldn't be
deleted before I re-download it. Any errors you see on it by the way? I
really don't like burdening you with that task. Currently trying to set up
the connection to the database so I can help with the SQL functions.

On Fri, Feb 17, 2017 at 3:44 AM, Justin Sexton <notifications@github.com>
wrote:

> If you're getting a sync error you may have to just delete the project and
> reclone it
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JSextonn/AgileRecords/pull/5#issuecomment-280604214>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIjVMtgFFcvxa-UW56M7W3S6FTO-kq4rks5rdWvigaJpZM4MEBBX>
> .
>



-- 
German Villalobos
german.villalobos0@gmail.com
